docs/sources/writing-guide/contribute-documentation/index.md

```diff
@@ -54,21 +54,45 @@ Small changes might include:
 
 > **Note:** You can only edit a topic that is part of the latest release of documentation.
 
-## Before you begin
+### Before you begin
 
 - Create a [GitHub](https://www.github.com) account.
 
 **To edit a topic:**
 
 1. From a topic on the documentation website, click **Edit this page** (pencil icon).
-2. Enter your changes.
-3. Change the branch name, if required.
+1. Enter your changes.
+1. Change the branch name, if required.
    The branch name is auto-populated.
 
-4. Click **Propose changes**.
+1. Click **Propose changes**.
 
    A PR is created and then goes through the PR review and approval workflow.
 
+## Develop a new topic
+
+If you want to develop a new topic from scratch, you can create a documentation plan and collaborate with a member of the Doc Squad to see it through. According to the book [_Docs for Developers_](https://docsfordevelopers.com/), a documentation plan is a _flexible outline_ for anticipating where the writing process will lead you.
+
+Your documentation plan will help you to:
+
+- Identify existing information gaps and explain how you will fill them
+- Get feedback from users and stakeholders before the writing process begins
+- Consider different approaches you might take, and decide between them
+
+> **Note:** You can only create new topics that are part of the latest release of documentation.
```

Comment:
What does this mean? I can create new topics for the `next` release of documentation. I rarely touch `latest` documentation.

Reply:
Do you mean "upcoming"?

Reply:
Technically, I can create new topics for any release of documentation, but it is not a good idea. :)

Reply:
I will delete the note.